core/jazz_ui/src/app/pages/service-overview/service-overview.component.html

```diff
@@ -314,7 +314,7 @@
             </li>
           </ul>
           <ul>
-              <li *ngIf="service.serviceType == 'function'">
+              <li *ngIf="service.serviceType == 'function' && false">
```

Comment:
Why "&& false"?

Reply:
right now we don't have this functionality but may be it will require in future